merge-bot.yml

```diff
@@ -0,0 +1 @@
+enabled: true
```

Comment:
Isn't that supposed to be in a .github subdir?

Reply:
Yup